flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -82,6 +82,10 @@
 
 	private static final long DEFAULT_RESTART_DELAY = 10000L;
 
+	// This field was used as a key for storing the EC in the Job Configuration
+	@Deprecated
+	public static final String CONFIG_KEY = "runtime.config";
```

Comment:
this change belongs in the "Fix breaking changes in flink-core" commit, unless they will be squashed.